extension/manifest.json

```diff
@@ -30,7 +30,7 @@
 		]
 	},
 	"permissions": ["storage", "notifications", "background", "https://api.torn.com/"],
-	"optional_permissions": ["https://www.tornstats.com/", "https://yata.yt/", "https://www.nukefamily.org/", "https://tornuhc.eu/"],
+	"optional_permissions": ["https://www.tornstats.com/", "https://yata.yt/", "https://www.nukefamily.org/", "https://tornuhc.eu/", "https://api.mavriware.me/"],

```

Comment:
This should be "https://api.mavriware.me/revive".

Reply:
No, the path doesn't matter for the permission.